### ISSUE TITLE
Outcome 4.3 - Capture household income from forest products

### ISSUE THREAD

**Issue by mruth-tesc** [Global-Evergreening-Alliance/Sandbox#5]
Average annual household income earned from sale of  timber and other non-timber forest products; % change in price per unit received by households from EverGreening value chains, disaggregated by product and gender.

**Comment by mruth-tesc** [comment]
List of non timber products (fodder, etc) - are these the same in every country (or do we have  different survey for each country? 
Fruit trees would not be included in this category?
What approaches are useful across the ARK countries, to capture price per unit?

**Comment by talialiney** [comment]
Non-timber products - at this point will be a generic list that should cover all locations

In regards to different species in different countries - I have suggested to Chris that we have different lists that will appear depending on the location chosen at beginning of survey.

At this stage specific income questions have been taken out of the baseline survey and a separate survey will be created - questions may include percentage of goods sold, consumed etc, estimation of income from different sources per month/year tbd. , number of units sold, plus more that aren't as relevant to forest products.

As soon as we have ability to share mindmap files, I will share my proposed survey structures
